File: Semester4/Systems for Design and Implementation/mpp-proiect-csharp-raulfrandes/networking/dtos/ResultDTO.cs
using System;

namespace networking.dtos;

[Serializable]
public class ResultDTO(long idParticipant, long idTrial, int points)
{
    public long IdParticipant { get; set; } = idParticipant;
    public long IdTrial { get; set; } = idTrial;
    public int Points { get; set; } = points;
}

File: Semester4/Systems for Design and Implementation/mpp-proiect-java-raulfrandes/TriathlonNetworking/src/main/java/triathlon/networking/protocol/dto/ResultDTO.java
package triathlon.networking.protocol.dto;

import java.io.Serializable;

public class ResultDTO implements Serializable {
    private Long idParticipant;
    private Long idTrial;
    private Integer points;

    public ResultDTO(Long idParticipant, Long idTrial, Integer points) {
        this.idParticipant = idParticipant;
        this.idTrial = idTrial;
        this.points = points;
    }

    public Long getIdParticipant() {
        return idParticipant;
    }

    public void setIdParticipant(Long idParticipant) {
        this.idParticipant = idParticipant;
    }

    public Long getIdTrial() {
        return idTrial;
    }

    public void setIdTrial(Long idTrial) {
        this.idTrial = idTrial;
    }

    public Integer getPoints() {
        return points;
    }

    public void setPoints(Integer points) {
        this.points = points;
    }
}
